build.sbt

```diff
@@ -59,6 +58,12 @@ lazy val root = (project in file("."))
     uglify / includeFilter  := GlobFilter("application.js")
   )
   .settings(CodeCoverageSettings.settings: _*)
+  .settings(
+    scalacOptions ++= Seq(
+      "-deprecation",
+      "-feature",
```

Comment:
Do we require another scalacOptions here ?
We can just add the "-deprecation" within lines 34 - 39 above, and not remove "-feature" from line 35